src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -61,14 +62,28 @@ def get(self, request):
 
 
 class OIDCAuthenticationCallbackView(_OIDCAuthenticationCallbackView):
+    def get_error_message_parameters(
+        self, parameter: str, problem_code: str
+    ) -> tuple[str, str]:
+        """
+        Return a tuple of the parameter type and the problem code.
+        """
+
+        return (parameter, problem_code)
+
     @property
     def failure_url(self):
         """
         On failure, redirect to the form with an appropriate error message
         """
         f = furl(self.success_url)
         f = furl(f.args["next"])
-        f.args[BACKEND_OUTAGE_RESPONSE_PARAMETER] = self.plugin_identifier
+
+        parameter, problem_code = self.get_error_message_parameters(
+            self.request.GET.get("error"), self.request.GET.get("error_description")
+        )
```

Comment:
```suggestion
        parameter, problem_code = self.get_error_message_parameters(
            self.request.GET.get("error", ""), self.request.GET.get("error_description", "")
        )
```
for type safety :) a bare `.get` returns `None` if the params are not present.